### ISSUE TITLE
Neighborhoods for lymph flow from organs > new lymph node compartment > venous blood?

### ISSUE THREAD

**Issue by eburg007** [Open-Systems-Pharmacology/MoBi#514]
Currently I am mapping neighborhoods as "organ_int_LymphNode _int" for each of the built in organs to illustrate lymph flow from organs to the lymph node. Then I have a final neighborhood "LymphNode_int_VenousBlood_pls" to illustrate lymph flow returning to blood plasma. Should I include any parameters as part of these neighborhoods (i.e. lymph flow rate)? 

Note: I have modified the lymph flow rate parameter within the lymph node organ to be calculated as Q*(1-HCT) (all other organ lymph flow rates are fractions of this total).

**Comment by StephanSchaller** [comment]
> Note: I have modified the lymph flow rate parameter within the lymph node organ to be calculated as Q*(1-HCT) (all other organ lymph flow rates are fractions of this total).

This term defines the plasma organ flow...
The lymph node to Venous Blood (total) lymph flow should equal the sum of all organ lymph flows...otherwise you will introduce and imbalance in your mass balance.
You can introduce a new parameter that sums them all up and use it in your new transporter (lymph to venous blood):
![image](https://user-images.githubusercontent.com/25477643/96265176-6b69c580-0fc5-11eb-9d93-24c072de614d.png)

Hope this helps

**Comment by eburg007** [comment]
Yes thanks! Also having trouble updating the building blocks of the simulation due to the error message for PassiveDiffusionInt2Cell that Partition Coefficient (interstitial/water) doesn't exist. I've seen previous posts about the issue with the calculation method supplying the formula that defines this parameter... how do I get around this?

**Comment by StephanSchaller** [comment]
You likely answered this?
This might help: https://github.com/Open-Systems-Pharmacology/Forum/discussions/881
